ZAPD/ZCollision.cpp

```diff
@@ -76,9 +77,25 @@ void ZCollisionHeader::ParseRawData()
 		polygonTypes.push_back(
 			BitConverter::ToUInt64BE(rawData, polyTypeDefSegmentOffset + (i * 8)));
 
-	if (camDataAddress != 0)
+	if (camDataAddress != 0) {
+		// Try to guess how many elements the CamDataList array has
```

Comment:
I think it's worth explaining what the order here is based on.

Reply:
I added an explanation. Tell me if it is clear enough

Reply:
I think probably the clearest explanation would involve giving the most common ordering.